python/taichi/misc/task.py

```diff
@@ -1,7 +1,36 @@
-from taichi.core import unit
+from taichi.core import util
+from taichi.misc.util import config_from_dict
 
 
-@unit('task')
+def _unit(unit_name):
```

Comment:
Does this mean unittest?

Reply:
`unit` was a wrapper over the legacy interface/impl-managing system used in legacy Taichi. I think there are still usages of `unit`. `task` is an example: https://taichi.readthedocs.io/en/stable/contributor_guide.html?highlight=libdevice#upgrading-cuda

The [interface](https://github.com/taichi-dev/taichi/blob/904181b5dc16c9d7d63974661befbae040300168/taichi/common/interface.h#L117)/unit system is intended to improve the modularity of the legacy Taichi library. I think we do not need them anymore. Given it seems overdesigned at this point, we may need to at least simplify it...

Reply:
Ah I see. 

> task is an example:

Right. I searched for `@unit` and `tasks` is the only user, so I moved `unit` here (instead of staying at `core`)..